tests/test_csc.py

```diff
@@ -37,29 +39,21 @@
 class CscTestCase(salobj.BaseCscTestCase, unittest.IsolatedAsyncioTestCase):
     def basic_make_csc(
         self,
-        initial_state=salobj.State.STANDBY,
-        config_dir=None,
-        simulation_mode=1,
-        **kwargs,
-    ):
+        initial_state: Optional[salobj.State | int]=salobj.State.STANDBY,

```

Comment:
Please do not use `Optional` anymore. Also do not use `salobj.State` but import `sal_enums` from ts_xml and use `sal_enums.State` instead. So this would simply become `sal_enums.State | int`. See https://docs.python.org/3.12/library/typing.html#typing.Optional